docs/operation/troubleshooting.md

```diff
@@ -0,0 +1,79 @@
+---
+id: troubleshooting
+title: Troubleshooting
+sidebar_label: Troubleshooting
+---
+
+This section contains information to help you identify and resolve errors you may encounter.
+
+### Known errors and issues
+
+#### lookup failed for scheduledEventID to activityID
```

Comment:
These are Go-specific errors from a Go-specific troubleshooting page and do not need to be in a general place